ProxyAppsSuite/tests/ember.yaml

```diff
@@ -0,0 +1,470 @@
+_base:
+    summary: Large scale modeling of high-performance interconnects
+    scheduler: slurm
+    slurm:
+        num_nodes: 2
+
+_build:
+    inherits_from: _base
+    build:
+        modules: [gcc]
+        source_path: ember
+        cmds:
+            # UCX
+            - 'module load friendly-testing openmpi/3.1.5'
+            - 'tar xvfz ucx-1.8.0.tar.gz &> ucx-build.log'
+            - 'cd ucx-1.8.0'
+            - 'UCX_PREFIX=${PWD}/../ucx_1.8.0'
+            - './configure CC=gcc CXX=g++ MPICC=mpicc --prefix=${UCX_PREFIX} --without-java'
+            - 'make -j'
+            - 'make install'
+            - 'cd ..'
+            # Openmpi
+            #- cp ../../openmpi-3.1.5.tar.bz2 .  || (echo "$0 at $LINENO FAILED"; exit 1;)
+            - 'bzip2 -d openmpi-3.1.5.tar.bz2'
+            - 'tar xvf openmpi-3.1.5.tar '
+            - 'module unload openmpi/3.1.5 '
+            - 'module load gcc '
+            - 'cd openmpi-3.1.5 '
+            - 'export LD_LIBRARY_PATH=${UCX_PREFIX}/lib:${LD_LIBRARY_PATH}'
+            - 'export PATH=${UCX_PREFIX}/bin:${PATH}'
+            - 'OMPI_PREFIX=${PWD}/../openmpi_3.1.5'
+            - 'find ../openmpi/3.1.5/platform -name "toss3-mlx-optimized.conf" '
```

Comment:
we should try to rename this file, and remove the other mpi version subdirs from the test_src before we put a big ribbon on this test.